charts/tempo-distributed/templates/memcached/_helpers-memcached.tpl

```diff
@@ -20,3 +20,19 @@ memcached selector labels
 {{ include "tempo.selectorLabels" . }}
 app.kubernetes.io/component: memcached
 {{- end }}
+
+{{/*
+memcached image
+*/}}
+{{- define "tempo.memcachedImage" -}}
+{{- $dict := dict "service" .Values.memcached.image "global" .Values.global.image "tempo" (dict) -}}
```

Comment:
The `tempoImage` template is looking for a tempo key in the provided dict, So, we could provide it with an empty dict.

Reply:
Let me describe it a little bit more here. from our last conversation, we decided to remove the `tempo` key from the `dict` object. 
I did that change then I got an error:
```text
UPGRADE FAILED: 
  template: tempo-distributed/templates/memcached/statefulset-memcached.yaml:41:20: 
  executing "tempo-distributed/templates/memcached/statefulset-memcached.yaml" at 
    <include "tempo.memcachedImage" .>: 
       error calling include: 
         template: tempo-distributed/templates/memcached/_helpers-memcached.tpl:29:4:
         executing "tempo.memcachedImage" at 
         <include "tempo.tempoImage" $dict>: error calling include: 
           template: tempo-distributed/templates/_helpers.tpl:31:67: 
           executing "tempo.tempoImage" at <.tempo.registry>: nil pointer evaluating interface {}.registry
```
The error says the `tempo.tempoImage` template in the `tempo-distributed/templates/_helpers.tpl` file needs a `tempo` key to find the `tempo.registry` 
in the given object (dict) but the `tempo` key has been removed.
As a resolution, we have to provide the `tempo` key in the `dict` object, and it has to be an empty object; hence I added the `tempo (dict)` to the `dict` object.

I hope that helped you to get to the point why the `tempo` key has to be set there.

Reply:
awesome, thank you!